numpy/linalg/tests/test_linalg.py

```diff
@@ -931,6 +932,14 @@ def test_empty_a_b(self, m, n, n_rhs):
         assert_equal(rank, min(m, n))
         assert_equal(s.shape, (min(m, n),))
 
+    def test_incompatible_dims(self):
+        # use modified version of docstring example
+        x = np.array([0, 1, 2, 3])
+        y = np.array([-1, 0.2, 0.9, 2.1, 3.3])
+        A = np.vstack([x, np.ones(len(x))]).T
```

Comment:
I'm somewhat tempted to change this (and the docstring) to `np.stack([x, np.ones(len(x))], axis=-1)